test/e2e/service.go

```diff
@@ -883,6 +888,32 @@ var _ = Describe("Services", func() {
 			}
 		}
 		validateUniqueOrFail(ingressPoints)
+
+		// This isn't actually part of the test, but it has the net effect of deleting the target pool and forwarding
```

Comment:
this won't run if any of the earlier assertions fail. maybe put this in an `afterEach` or `defer`red cleanup func so it always runs?